packages/sit-onyx/src/components/OnyxNavBar/OnyxNavBar.stories.ts

```diff
@@ -134,3 +136,40 @@ export const WithCustomAppArea = {
     appArea: () => [h(OnyxIcon, { icon: placeholder, color: "secondary" }), "Custom name"],
   },
 } satisfies Story;
+
+/**
+ * This example should be watched at a mobile breakpoint.
+ * Both the nav area as well as the context area will overflow.
+ */
+export const WithOverflowingMobileContent = {
```

Comment:
The example does not look right on desktop
![image](https://github.com/SchwarzIT/onyx/assets/67898185/ab22a012-1c20-45c1-a3de-625f8743d81e)

Reply:
yes, we don't have the more feature yet. pls see the comment on top of this example that says: please look at the mobile breakpoint with this example
- #986

Reply:
Hmm should we maybe increase the mobile breakpoint then for this example?
As a user, looking at this seems broken imho

Reply:
ah right, we can do this! :)
I'll look up how and do that ^^

Reply:
done :)